packages/config/config/devices/0x013c/psk01.json

```
@@ -0,0 +1,73 @@
+{
+	"manufacturer": "Philio Technology Corp",
+	"manufacturerId": "0x013c",
+	"label": "PSK01",
+	"description": "Smart Keypad",
+	"devices": [
+		{
+			"productType": "0x000a",
+			"productId": "0x006f"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "~/templates/master_template.json#base_options_nounit",
+			"label": "Set Keypad Mode",
+			"description": "Change between Entry control mode and Central Scene mode.",
+			"defaultValue": 2,
+			"options": [
+				{
+					"label": "Entry control mode",
+					"value": 1
+				},
+				{
+					"label": "Central Scene mode",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Buzzer",
+			"description": "Enable or disable the buzzer.",
+			"defaultValue": 1
+		},
+		{
+			"#": "3",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Rand Mode",
+			"description": "When the Rand Mode is enabled, whenever the device wakes up from sleeping, no more than four digits of random keys will light on. Users have to press those random keys one by one, and the device will go to Entry Control mode."
+		},
+		{
+			"#": "4",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Touch Sleep",
+			"description": "If Touch Sleep is enabled the device will go to sleep a short time after the last touch.",
```

The device doesn't actually go to sleep it seems. It's just the panel lights that turn off until the next time the panel is touched.
```suggestion
			"description": "If Touch Sleep is enabled the touch panel lights will turn off a short time after the last touch.",
```